README.md

```diff
@@ -1,2 +1,78 @@
+# Overview
```

Comment:
I think it would be sufficient to provide a link to the docs instead of this whole overview as is, and keep this README cleaner and more straightforward, as this is the README for the charms, not the product itself.
As @sabaini  suggests, we need to answer important questions e.g. What is it? What does it do? What essential problem does it solve? Who is it useful for? 
The next question seems to attempt to answer these questions, but I suggest we rename it to something more descriptive, e.g. "Ceph charms Monorepo" and maybe include a small background/explanation and remove the "Overview" and "Repository contents" section.